src/toc.md

```diff
@@ -3,5 +3,6 @@
 - [Introduction](index.md)
 - [Documentation](documentation/index.md)
```

Comment:
> I don't believe this should be under the Documentation group, as that is for documentation about our documentation system. Perhaps this should be retitled as "Documentation System" to make this clearer?

Agreed. Should we make a new section "Workflows" then and place it there?

> Regarding the file itself, I think this makes sense being that we have no other data about limitations or the Collector-to-Survey123 workflow, however I do wonder if we should actually make a limitations.md document and put this information under a ## Collector to Survey123 Workflow section. I think the answer will be based on "Do we plan on adding more information about the workflow or more information about limitations?"

My thinking was that we will eventually transfer Chandra's documentation about the workflow into this document and we document the limitations with the workflow it affects. 

> I bring this up because I believe that one of our partners are experiencing a bug due to the version of Android they are using and I'm wondering where that will be documented internally.

If I read this correctly, what you're suggesting is a _Troubleshooting_ page (i.e. "Look here if something doesn't work, maybe we already have a solution"). I think that makes sense for the more generic bug you have outlined above. Downloading images for offline use isn't a limitation related to any specific workflow but the limitation with the URLs linking from Collector to the white-labeled Survey123 is. That's why I'd put the latter as a limitation with the workflow itself. 

However, I can see that both ways make sense, whichever way you look at it. We should ask programs where they would look for this sort of information.